README.md

```diff
@@ -1 +1,296 @@
+
+- **[See overview of m]**
+  - Rent a house .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Link](https://cashverse-raga.onrender.com/)
+- [Loom video](https://www.loom.com/share/fa994de927394858b0a81204dafdb1c4)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd folder
+  git clone 
+  cd budget-app
+  Open in your favorite editor
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd https://github.com/zakiyousufi/Rental-House-API.git
+  gem install
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  bin/rspec spec
+```
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **zakiyousufi Ahmad Zaki Yousufi**
+
+- GitHub: [](https://github.com/zakiyousufi)
+- Twitter: []()
+- LinkedIn: []()
+## 👥 Authors <a name="authors"></a>
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **UsirusenIkon Usirusen Ikon**
+
+- GitHub: [](https://github.com/UsirusenIkon)
+- Twitter: []()
+- LinkedIn: []()
+
+
+
+👤 **Profsain **
+
+- GitHub: []()
+- Twitter: []()
+- LinkedIn: []()
+## 👥 Authors <a name="authors">Profsain </a>
+👤 **soulemanou kunchiefuh **
+
+- GitHub: []()
+- Twitter: []()
+- LinkedIn: []()
+## 👥 Authors <a name="authors">Profsain </a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[List Transactions]**
+- [ ] **[Transaction Details]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
Kindly add two or more Acknowledgments 
You can mention `Microverse, Code reviewers, Coding partner etc`